config/clients/go/template/client/client_test.mustache

```diff
@@ -1367,7 +1474,7 @@ func Test{{appShortName}}Client(t *testing.T) {
 			t.Fatalf("%v", err)
 		}
 
-		if httpmock.GetTotalCallCount() != 4 {
+		if httpmock.GetTotalCallCount() != 5 {
 			t.Fatalf("{{appShortName}}Client.%v() - wanted %v calls to /check, got %v", test.Name, 4, httpmock.GetTotalCallCount())
```

Comment:
The error message here is now incorrect, it's 5 calls to check and 1 to check valid authorization model

Reply:
fixed